int32 selectedUAV
int32 selectedUAV_mode
